src/object-store/src/shared_realm.hpp

```diff
@@ -82,7 +82,10 @@ namespace realm {
             // everything can be done deterministically on one thread, and
             // speeds up tests that don't need notifications.
             bool automatic_change_notifications = true;
-
+            // File format versions populated when a file format updrade takes place
+            // during realm opening
+            int upgrade_initial_version, upgrade_final_version;
```

Comment:
You could just default each of these to `0` here, and remove them from the initializer lists of the constructors.